hack/pre-run-e2e.sh

```diff
@@ -15,16 +15,17 @@
 # limitations under the License.
 
 set -o errexit
-set -o nounset
 set -o pipefail
 
 if [[ "${HC_ACCESS_KEY}" = "" ]] || [[ "${HC_ACCESS_KEY}" = "" ]]; then
   echo "HC_ACCESS_KEY and HC_ACCESS_KEY can not be empty, please set it"
   exit 1
 fi
+set -o nounset
```

Comment:
I suppose you mean the environment `HC_ACCESS_KEY` always not exist. 
Then, why not just remove line [20-23]? They are redundant, isn't ?

Reply:
OK. Since the environment is well-configured, so, i guess there will be no errors, 
but why did you move the line(`set -o nounset`) down?

Reply:
because `HC_ACCESS_KEY` is undefined in the script, it will cause an error and exit.
Move down `set -o nounset` to skip this checking validation.
This is a temporary solution, it will be optimized later.